paddle/fluid/framework/new_executor/program_interpreter.h

```diff
@@ -81,6 +81,8 @@ class ProgramInterpreter : public InterpreterBaseImpl {
     hookfuncs_ = hookfuncs;
   }
 
+  bool IsRealStaticBuild() const { return static_build_; }
```

Comment:
```suggestion
  bool IsStaticBuild() const { return static_build_; }
```
RealStaticBuild的命名容易让人误解，建议直接和成员变量static_build_保持一致命名。

Reply:
Done.Thanks!